README.md

```diff
@@ -0,0 +1,30 @@
+# Learning Journal
+
+A React Native/GraphQL based application for running 'Learning Journal'
+
+## Description
```

Comment:
Maybe place the description under Learning Journal and remove the description title.

Reply:
It doesn't make a lot of difference in my opinion. I think in v2 of this we refine it more, add screenshots/animations of the app as well. We can then make more changes. What do you say?